db/migrate/20151210225310_create_test_client_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTestClientRequests < ActiveRecord::Migration
```

Comment:
This implementation is supposed to *be* a "generic client data object".

How are you imagining a generic client data object would look?